### ISSUE TITLE
2.4.x + Ilford + spring.cloud.config.tls.enabled=true

### ISSUE THREAD

**Issue by patpatpat123** [spring-cloud/spring-cloud-config#1794]
Hello Team, 

This following PR:
https://github.com/spring-cloud/spring-cloud-config/pull/1679/files
Is a great feature and allow mTLS between Spring Cloud Config Server and Client (and even having to override a custom rest template beans or ConfigServicePropertySourceLocator bean.

It is working fine with 2.3.x

With the breaking change introduce with 2.4, it seems those properties are not taken effect.

Before: 2.3.x + Hoxton.SRx + properties in bootstrap = 😃
After: 2.4.x + Ilford + properties in application.yml = 😭

The properties are:
```
spring.cloud.config.tls.enabled=true
spring.cloud.config.tls.key-store=/path/to/my/keystore
(and the two passwords for key/keystore)
```

May I as kid it is a bug introduced by the breaking change from 2.4 please? (Or simply, I misunderstood and those are not the correct properties anymore?

Thank you

**Comment by spencergibb** [comment]
The current workaround is to add `spring-cloud-starter-bootstrap`. This was not added due to time constraints.

Also see https://github.com/spring-cloud/spring-cloud-config/issues/1689

**Comment by patpatpat123** [comment]
Many thanks. @spencergibb 

Closing in favor of https://github.com/spring-cloud/spring-cloud-config/issues/1689
